From foo@baz Fri Nov 21 10:58:10 PST 2008
Date: Fri, 21 Nov 2008 10:58:10 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: comedi: fix checkpatch.pl warning in interrupt.h

No more need for a kernel version check now that we are in the main
kernel tree.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/interrupt.h |    7 -------
 1 file changed, 7 deletions(-)

--- a/drivers/staging/comedi/interrupt.h
+++ b/drivers/staging/comedi/interrupt.h
@@ -53,15 +53,8 @@ typedef void irqreturn_t;
 #endif
 #endif
 
-/* if interrupt handler prototype has pt_regs* parameter */
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 19)
-#define PT_REGS_ARG , struct pt_regs *regs
-#define PT_REGS_CALL , regs
-#define PT_REGS_NULL , NULL
-#else
 #define PT_REGS_ARG
 #define PT_REGS_CALL
 #define PT_REGS_NULL
-#endif
 
 #endif
